Client/Frontend/Browser/TabTrayButtonExtensions.swift

```diff
@@ -4,23 +4,30 @@
 
 import UIKit
 
-class PrivateModeButton: ToggleButton, Themeable {
+class PrivateModeButton: InsetButton, Themeable {
     var light: Bool = false
     
+    override var isSelected: Bool {
+        didSet {
+            backgroundColor = isSelected ? UIColor.Photon.Purple60 : .clear
+        }
+    }
+    
     override init(frame: CGRect) {
         super.init(frame: frame)
         accessibilityLabel = PrivateModeStrings.toggleAccessibilityLabel
         accessibilityHint = PrivateModeStrings.toggleAccessibilityHint
-        let maskImage = #imageLiteral(resourceName: "smallPrivateMask").template
-        setImage(maskImage, for: [])
+        
+        titleEdgeInsets = UIEdgeInsets(top: -3, left: 6, bottom: -3, right: 6)
+        layer.cornerRadius = 4.0
```

Comment:
nit: could be a constant so that the same corner radius can be used throughout the ap

Reply:
From what I've checked there's only a few other corner radius with the same value of 4, and the rest are either literals in place or local UX-esque structs, but if we're only going to have this one thing in it I'd rather keep it as a literal

Reply:
👍